### ISSUE TITLE
Linter/syntax checker

### ISSUE THREAD

**Issue by vikrambombhi** [jaydenwindle/decode2017#61]
## Expected Behavior
We should get a linter/syntax checker. It seems like alot of the requested issues on PR's are small syntax issues. Basic syntax things like spacing and semicolons can be taken care of by a linter and we can concentrate our reviews on the logic of the code. 

## Current Behavior
Most PR reviews are syntax change requests not logic.

## Possible Solution
Some sort of linter. Im sure rails has something like this but we have to find it/decide on which one to use.

**Comment by jaydenwindle** [comment]
Good call. Is there a style guide check / linter we can add to our travis ci?

**Comment by karimsa** [comment]
I normally do this via node.js, but there's probably some sort of bridge to set it up for rails.

I'll have a look soon.
